README.md

```diff
@@ -12,22 +28,22 @@
 
 Running this app is as simple as deploying it to a balenaCloud fleet. You can do it in just one click by using the button below:
 
-[![deploy button](https://balena.io/deploy.svg)](https://dashboard.balena-cloud.com/deploy?repoUrl=https://github.com/balena-io-experimental/balena-sound&defaultDeviceType=raspberry-pi)
+[![deploy button](https://balena.io/deploy.svg)](https://dashboard.balena-cloud.com/deploy?repoUrl=https://github.com/iotsound/iotsound&defaultDeviceType=raspberry-pi)
 
 ## Documentation
 
-Head over to our [docs](https://balena-sound.pages.dev) for detailed installation and usage instructions, customization options, and more!
+Head over to our [docs](https://iotsound.github.io/iotsound/) for detailed installation and usage instructions, customization options, and more!
```

Comment:
```suggestion
Head over to our [docs](https://iotsound.github.io/balena-sound/) for detailed installation and usage instructions, customization options, and more!
```

@alanb128 To preserve the redirect, the repository would be named balena-sound so the URLs would be https://iotsound.github.io/balena-sound/ and https://github.com/iotsound/balena-sound/